CHANGELOG.rst

```diff
@@ -79,6 +79,20 @@
 
 *
 
+**Changes**
```

Comment:
Deliberately placed it here because this should be moved to the appropriate section on `features`.